### PR TITLE
DOC document supported upload formats

### LINKED ISSUE #489: More documentation of formats on upload page
For each upload format, we should:
* reference its correct name and perhaps documentation, e.g. "MS COCO Format", "PASCAL VOC XML"
* specify which task types it supports (classification, bounding box or segmentation)

This could be done by adding a table in the /upload page

### PR DESCRIPTION
**Pull request by jnothman**
Fixes #489

### CODE REVIEW

File: search/src/Components/wrapper/upload.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
 ## Annotation Formats
```
Comment (jnothman) [inline search/src/Components/wrapper/upload.md:1]: @geezacoleman would you mind checking the changes to this file? Thanks

File: search/src/Components/wrapper/upload.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+| Pascal VOC XML | | ✔️ | |
+| Color-coded segmentation masks | | | ✔️ |
 
 We natively support WeedCOCO format which extends on MS COCO to specify a weed
 ID-oriented category naming scheme, to include agricultural context and
-[schema.org/Dataset](https://schema.org/Dataset)-compatible metadata. We provide an uploader
-for MS COCO format, with forms to enter agricultural context and metadata
-please ensure the category names are conformant before uploading.
+[schema.org/Dataset](https://schema.org/Dataset)-compatible metadata.
+Uploads in other formats be converted to WeedCOCO format, with
```
Comment (geezacoleman) [inline search/src/Components/wrapper/upload.md:15]: > Uploads in other formats *can* be converted to...

File: search/src/Components/wrapper/upload.md [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
+forms to enter agricultural context and metadata, as well as to
```
Comment (geezacoleman) [inline search/src/Components/wrapper/upload.md:16]: Probably worth linking those here: [agricultural context](https://weed-ai.sydney.edu.au/editor) and [metadata](https://weed-ai.sydney.edu.au/meta-editor)

File: search/src/Components/wrapper/upload.md [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
+map category names to our standardised nomenclature.
 
 We require that contributors license their images and annotations under the
 liberal [CC-BY 4.0 licence](https://creativecommons.org/licenses/by/4.0/)
```
Comment (geezacoleman) [inline search/src/Components/wrapper/upload.md:20]: I would remove 'the liberal'

File: search/src/Components/wrapper/upload.md [hunk 5]
```diff
@@ -20,6 +27,7 @@ users to freely share and adapt the material for any purpose, even
 commercially, given appropriate attribution and that there are no additional
 restrictions made.
```
Comment (geezacoleman) [inline search/src/Components/wrapper/upload.md:30]: Should add something along the lines of (taken from creative commons website): "This is a human-readable summary of (and not a substitute for) the license".

File: search/src/Components/wrapper/upload.md (hunk 1 above)
Comment (geezacoleman) [reply search/src/Components/wrapper/upload.md:1]: Only some minor changes suggested below

**Review by geezacoleman — COMMENTED** [review]
Suggested changes as specified in the comments.

**Comment by jnothman** [comment]
Thanks!

**Pull request merged**